src/test/java/htsjdk/samtools/cram/io/IOTestCases.java

```diff
@@ -140,6 +157,28 @@
         };
     }
 
+    // for testing ITF8 and LTF8 equivalence: only true in the range 0 - (high bit = 28)
```

Comment:
Same here, expand a bit on this comment to explain why these tests cases are included.